src/main/resources/assets/react/collection/useCollectionState.ts

```diff
@@ -162,6 +164,46 @@ export function useCollectionState({
 		validateName
 	]);
 
+	//──────────────────────────────────────────────────────────────────────────
+	// Init and updates
+	//──────────────────────────────────────────────────────────────────────────
+	React.useEffect(() => {
+		if (collectorName) {
+			fetch(`${servicesBaseUrl}/graphQL`, {
+				method: 'POST',
+				headers: { 'Content-Type': 'application/json' },
+				body: JSON.stringify(gql.query({
+					operation: 'getManagedDocumentTypes',
+					variables: {
+						collectorId: {
+							required: true,
+							type: 'ID',
+							value: collectorName
+						}
+					}/*,
+					fields: []*/
+				}))
+			})
+				.then(response => response.json())
+				.then(json => {
+					// console.debug('json', json)
+					const {
+						data: {
+							getManagedDocumentTypes
+						}
+					} = json;
+					// console.debug('setting managedDocumentTypes to', getManagedDocumentTypes);
+					setManagedDocumentTypes(getManagedDocumentTypes);
+				});
```

Comment:
No error handling here? What if request fails?

Reply:
Yeah, we really don't have much error handling in the frontend. We would need some way of showing errors to the user.